package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@jinntec/fore",
   "version": "0.11.1",
   "description": "Fore - Forms for the Web",
-  "module": "./index.js",
```

Comment:
not sure here - doesn't that mean that i publish the bundle to npm when doing npm publish? I'd like to keep the unbundled npm package (as: optimization is application responsibility)

Reply:
Agreed, I'll add it to the files instead

Reply:
Done, it is added to "files"